mainhub/mainhub.ino

```diff
@@ -93,19 +117,63 @@ void update_game()
     cursor_index = 1; // Move to the second LED
     direction = 1; // Change the direction to right
   }
-  // Increment the cursor, but reset to zero if it is greater than the number of leds (Circle Path)
-  /*cursor_index = cursor_index + 1;
-  if (cursor_index >= NUM_LEDS)
-  {
-    cursor_index = 0;
-  }
-*/
+  //LEDs
+  //Light the cursor
+  leds[cursor_index] = rainbowColors[currentLevel];
+  FastLED.show();
+  // Now turn the LED off, then pause
+  leds[cursor_index] = CRGB::Black;
+  FastLED.show();
```

Comment:
I love this blink effect. 

Another way to do a blink is to have a variable called `int blink_mode;` which gets sets based on a timegate and then you use that to draw

e.g. 

```
if (blink_mode){
   leds[cursor_index] = rainbowColors[currentLevel];
else {
   leds[cursor_index] = CRGB::Black;
}
```

This would allow you to control how fast the blink happens and keep the system running fast